operator/api/v1alpha1/wavefront_types.go

```diff
@@ -187,6 +190,15 @@ type WavefrontProxy struct {
 
 	// ProxyVersion is for internal use only
 	ProxyVersion string `json:"-"`
+
+	// PreprocessorRules is for internal use only
+	PreprocessorRules PreprocessorRules `json:"-"`
```

Comment:
looks like we have a pattern here. Could we have an internal struct which encompasses all such structs/parameters.
I feel accessing will be much easier as DataExport.Metrics.Internal.PreprocessorRules , to designate its being mutated in operator code (could definitely be a separate change), any thoughts.